kafka-connect-s3/src/test/java/io/confluent/connect/s3/DataWriterJsonTest.java

```diff
@@ -46,8 +46,8 @@
 import io.confluent.connect.storage.partitioner.Partitioner;
 
 import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
please add a comment when the reason of the change is not obvious

Reply:
mentioned in the PR description, this fixes deprecation warnings